### PR TITLE
Feature - deleting bloat files

### PR DESCRIPTION
**Pull request by visill**
New sql func yezzey_vacuum_garbage_relation('<table_name>',bool confirm,bool crazyDrop);
- Removes bloat files by table name

New internal c funcs
- yezzey_vacuum_garbage_relation_internal(Re rel / OId reloid, int segindx, bool confirm, bool crazyDrop)
- - Deletes bloat files from a table by relation or oid

Slightly rewritten vacuum hook (does not worked correct) ((does not work))

https://github.com/open-gpdb/yproxy/pull/72

### CODE REVIEW

File: src/xvacuum.cpp [hunk 1]
```diff
@@ -79,3 +79,51 @@ int yezzey_vacuum_garbage_internal(int segindx, bool confirm, bool crazyDrop) {
...
+
+    auto tp = SearchSysCache1(NAMESPACEOID,
+                              ObjectIdGetDatum(RelationGetNamespace(rel)));
+
+    if (!HeapTupleIsValid(tp)) {
+      elog(ERROR, "yezzey: failed to get namescape name of relation %d",
+           RelationGetNamespace(rel));
+    }
+    
+    relnodeCoord coords{1663,rel->rd_node.dbNode,rel->rd_node.relNode,segindx};
```
Comment (diPhantxm) [inline src/xvacuum.cpp:102]: 1663 -> rel->rd_node.spcNode ??

**Review by reshke — APPROVED** [review]

File: src/xvacuum.cpp (hunk 1 above)
Comment (visill) [reply src/xvacuum.cpp:102]: rel->rd_node.spcNode  == 8555

Comment (visill) [reply src/xvacuum.cpp:102]: == YEZZEYTABLESPACE_OID

Comment (visill) [reply src/xvacuum.cpp:102]: 1663 == DEFAULT_TABLESPACE

Comment (visill) [reply src/xvacuum.cpp:102]: https://github.com/open-gpdb/yezzey/pull/56

**Pull request merged**